chart/assets/operations/instance_groups/locket.yaml

```diff
@@ -0,0 +1,43 @@
+# Set instance count to 2; the default inherited from diego-api is 2
```

Comment:
This is not necessary; the `_jobs.move` logic will do this automatically: https://github.com/cloudfoundry-incubator/kubecf/blob/9efc4b3/chart/templates/_jobs.tpl#L178

More generally, the new instance group also needs to be hooked up to the "sizing" mechanism in https://github.com/cloudfoundry-incubator/kubecf/blob/master/chart/assets/operations/sizing.yaml

So there should also be an entry for `sizing.log_cache.instances: ~` in `values.yaml`.

Finally, I think that with this change `diego-api` is no longer needed when running with Eirini, so please update the various places where `diego-cell` is disabled and tread `diego-api` similarly. Or tell me why it is still needed even after locket has been moved out.

Maybe also add `Fixes #1410` to the PR description to make sure the issue gets closed when this PR gets merged?

Reply:
ok working on this thanks.